source/MRMesh/MRMeshLoadObj.cpp

```diff
@@ -813,19 +821,6 @@ Expected<std::vector<NamedMesh>> fromSceneObjFile( const char* data, size_t size
         case ObjElement::Unknown:
             break;
         case ObjElement::Vertex:
-            if ( settings.customXf && !pointOffset.has_value() )
-            {
-                auto res1 = parseVertex( group.begin );
-                if ( !res1.has_value() )
-                {
-                    parseError = std::move( res1.error() );
-                    break;
-                }
-                auto [v, c] = *res1;
-                pointOffset.emplace( v );
-                if ( c != cInvalidColor )
-                    hasColors = true;
```

Comment:
That's the only place the variable is set; restore the code block to preserve the color support.